src/utils.rs

```diff
@@ -145,18 +148,61 @@ pub fn get_url_type(dependency_url: &str) -> UrlType {
     UrlType::Http
 }
 
+pub fn sanitize_dependency_name(dependency_name: &str) -> String {
+    if FILE_NAME_REGEX.is_match(dependency_name) {
+        return String::new();
+    }
+    dependency_name.replace("/", "-")
+}
```

Comment:
A bit simplistic, and we could avoid to error if we instead replace the offending characters by allowed ones.
Maybe we can use https://crates.io/crates/sanitize-filename instead of rolling our own. They handle windows special chars and max file length too.